actnow/petitions/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from actnow.accounts.models import ActNowUser
+from actnow.db.models import TimestampedModelMixin
+
+
+class Petition(TimestampedModelMixin):
+    title = models.CharField(
+        _("title"),
+        max_length=255,
+    )
+    description = models.CharField(
+        _("details of petition"),
+        max_length=1024,
+        blank=True,
+    )
+    owner = models.ForeignKey(
+        ActNowUser,
+        on_delete=models.SET_NULL,
+        null=True,
+    )
+    recipients = models.CharField(
+        _("recipients"),
+        max_length=255,
+    )
+    problem_statement = models.CharField(
```

Comment:
this will be approximately 200-250 words. Is that the maximum we can have or should we use a TextField?